### ISSUE TITLE
Default templates in docs aren't up to date

### ISSUE THREAD

**Issue by railapex** [eduludi/jquery-yacal#3]
Great control, btw!

Just ran into this today, though. I copy pasted default templates from https://github.com/eduludi/jquery-yacal#templates and started tweaking them. However, the current code is using slightly different ones. Not sure which are right, so I didn't do a PR for this.

**Comment by eduludi** [comment]
Did you refer to "monthNumber" / "month"? There is another one? Thanks for your feedback on this.

**Comment by railapex** [comment]
The one that jumped out at me was the day one:

In the code it is:

``` javascript
'<a  class="day d#dayWeek##weekend##today##selected##active#" data-time="#time#"><span>#day#</span></a>'
```

But the docs show it as:

``` javascript
'<a class="day d#weekday##weekend##today##selected#" href="##timestamp#">#day#</a>'
```
- dayWeek vs. weekday
- href vs. data-time
- timestamp vs time

If the code is the right one, somebody probably needs to go through the docs and copy/paste across what's in the code to the docs. Because there were so many differences I didn't want to just PR without knowing what the right one was.

**Comment by eduludi** [comment]
Ok. I fixed all of them. And now  **dayWeek** was removed, as _weekday_ does the trick for both now.

**Comment by eduludi** [comment]
Fixed in c7b3d3ced8b6b41965b602d5e83d4541980213ea
